roles/baseline/defaults/main.yml

```diff
@@ -25,12 +25,25 @@ METRICS_SERVER_CONFIG:
   apiService:
     create: true
 
+## Ingress-nginx - Defaults
+ingressVersions:
+  k8sMinorVersionCeiling:
+    value: 21
+    api:
+      chartVersion: 3.20.1
+      appVersion: 0.43.0
```

Comment:
Don't mind having these items, but they are really not relevant. The appVersion is controlled by the chartVersion selected. Just extra info or something to keep track off.